visma/simplify/simplify.py

```diff
@@ -259,14 +260,24 @@ def expressionSimplification(tokens_now, scope, tokens1):
     if expressionPresent:
         animation += [simToks]
         comments += [['Opening up all the brackets']]
-    if scope == []:
-        simToks, availableOperations, token_string, animExtra, commentExtra = simplifification(simToks)
-        animExtra.pop(0)
-        animation += animExtra
-        comments += commentExtra
+
+    trigonometricError = False
```

Comment:
What is this trigonometric error check for?

Reply:
The simplify module is not able to simplify the terms involving Trigonometric Expressions (like `sin(x)*x`. This has been marked as a TODO for now. When boolean `trigonometricError` turns true (i.e. some trigonometric term is there in the input), these tokens are left untouched by simplifying module.